### DISCUSSION TITLE
Apply org-mode in company doc-buffer not fontified.

### DISCUSSION THREAD

**Discussion by stardiviner** [company-mode/company-mode#1265] [Q&A] [answered]
I try to set company doc-buffer to org-mode, but it's not fontified as org-mode color, It's just black color for text.

Here is my code https://github.com/stardiviner/org-contacts.el/blob/main/org-contacts.el#L660

Does anybody know why?

**Comment by dgutov** [comment]
How are you viewing the said doc buffer? With `C-h`?

**Reply by stardiviner** [reply]
I use custom keybinding to command `(define-key company-active-map (kbd "M-h") 'company-show-doc-buffer)`.

**Reply by stardiviner** [reply]
It looks like this:

![image](https://user-images.githubusercontent.com/378699/143170188-7060cd24-69a8-45d9-895d-bf1e6c1aad84.png)

**Reply by dgutov** [reply]
Which part of the screenshot shows the doc-buffer? (In org-mode, not syntax highlighted).

**Reply by stardiviner** [reply]
The `*org-contact*` part is the doc-buffer. Sorry about not marked the doc-buffer in screenshot.

**Reply by dgutov** [reply]
Hm, okay, thanks.

Can you try adding `(font-lock-ensure)` after `(org-show-all)`?

**Reply by stardiviner** [reply]
Problem solved. Thanks a lot @dgutov .

I actually have thought about this direction and tried with `(font-lock-mode)`. But it's not working.

**Comment by stardiviner** [comment, marked as answer]
Adding `(font-lock-ensure)` after `(org-show-all)`.
